docs/Using-WDT.md

```diff
@@ -29,17 +30,17 @@ If the sample git repository hasn't been cloned yet, WDT has git tools integrate
 
 This sample can be built using [Maven](#building-with-maven).
 
-
 #### Building with [Maven](http://maven.apache.org/)
 
 ###### Import Maven projects into WDT
 
-1.  In the Git Repository view, expand the sample repo to see the "Working Directory" folder
+1.  In the Git Repository view, expand the sample repo to see the "Working Tree" folder
```

Comment:
Maybe before this list, we should say something like:  

These instructions are written for WDT 17.0.0.2 or later that has improved Maven project integration.